README.md

```diff
@@ -79,6 +84,11 @@ sending as fast as the code allows.
   malaria publish -P 8 -n 10000 -H mqtt.example.org -s 100
 ```
 
+For doing the same as above, now using the containerized version:
+```
+  docker run flaviostutz/mqtt-malaria publish -P 8 -n 10000 -H mqtt.example.org -s 100
```

Comment:
same here, either convert this to syntax for a locally built docker, or drop it entirely, require container people know what they're doing.